ehrapy/anndata/_feature_specifications.py

```diff
@@ -0,0 +1,97 @@
+from typing import Literal
+
+import numpy as np
+import pandas as pd
+from rich import print
+from rich.tree import Tree
+
+from ehrapy import logging as logg
+from ehrapy.anndata._constants import CATEGORICAL_TAG, CONTINUOUS_TAG, DATE_TAG, FEATURE_TYPE_KEY
+from ehrapy.anndata.anndata_ext import anndata_to_df
+
+
+def infer_feature_types(adata, layer: str | None = None, output: Literal["print", "dataframe"] | None = "print"):
+    """
+    Infer feature types from AnnData object.
+
+    For each feature in adata.var_names, the method infers one of the following types: 'date', 'categorical', or 'continuous'.
+    The inferred types are stored in adata.var['feature_type']. Please check the inferred types and adjust if necessary using
```

Comment:
This might require more explanation of what to do with 
"1", "0" & friends. We should be clear to the user whether these should be categorical or numerical because it's not completely obvious